app/src/main/AndroidManifest.xml

```diff
@@ -1,19 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="mk.edu.uklo.fikt.fiktexam" >
+    package="mk.edu.uklo.fikt.fiktexam">
 
     <uses-permission android:name="android.permission.INTERNET" />
-    
+
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name"
```

Comment:
It is not a good idea to hardcore text in the Manifest, do the text change in the strings.xml